components/servo/fake-ld.sh

```diff
@@ -9,7 +9,23 @@ set -o nounset
 set -o pipefail
 
 TARGET_DIR="${OUT_DIR}/../../.."
-arm-linux-androideabi-gcc "${@}" \
-                          "${LDFLAGS-}" -lc -shared \
-                          -o "${TARGET_DIR}/libservo.so"
-touch "${TARGET_DIR}/servo"
+
+export _ANDROID_ARCH=arch-arm
+export ANDROID_SYSROOT="$ANDROID_NDK/platforms/android-18/$_ANDROID_ARCH"
```

Comment:
Please use the more secure form `"${ANDROID_NDK}/platforms...."` for shell variables. This also applies to other variables in this patch. (See https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md)